htdocs/ticket/card.php

```diff
@@ -244,7 +244,9 @@
 
 			if (!$error) {
 				// File transfer
-				$object->copyFilesForTicket();
+				if ($object->notify_tiers_at_create) {
```

Comment:
Without this PR, when you create a new ticket from backoffice, and checkbox "send notification at creation" is checked, there is already an email sent to thirdparty.
Does this mean here you send 2 email to third party ?